rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [x] Since the application expects us to read and write all these properties, let's make all of them readable and writable. The next review thread will tell us why they should be all readable and writable.
```suggestion
  attr_accessor :date, :book, :person
```

Reply:
I also have a doubt here: if I implement the setters for `book` and `person` that way, it would be allowed to change those values without implementing the required logic to modify the `rentals` lists in the affected `book`s or `person`s. Is that okay?

Reply:
It's okay. This is because when will be creating a rental all these will need to be supplied as arguments. Therefore, making them readable and writable is neccessary.

Reply:
Ah thank you for the explanation! I thought the `attr_accessor` was only to create automatically the default getter and setter.

Reply:
You are right and we are all saying the same thing. Setter means writable, getter means, readable. Initially you made them only readable but some other entity would actually need to write them which would have made the app buggy.